packages/web-components/fast-components/src/tree-item/tree-item.styles.ts

```diff
@@ -210,7 +218,9 @@ export const treeItemStyles: FoundationElementTemplate<ElementStyles, TreeItemOp
     }
 
     :host([expanded]) > .items {
-        display: block;
```

Comment:
Per the above comment, this is already a block element - with the removal of above, this becomes redundant.